dbt/models/reporting/schema.yml

```diff
@@ -1,10 +1,16 @@
 sources:
   - name: reporting
     tags:
+      - daily
       - load_auto
     tables:
       - name: ratio_stats
         description: '{{ doc("table_ratio_stats") }}'
+        config:
+          packages:
+            - "assesspy==1.1.*"
+            - "numpy==1.26.*"
+            - "pandas==2.*"
```

Comment:
This is the "dbt way" to define package dependencies. I'm not yet clear on exactly how we'd use it in the package tooling. My first pass idea:

1. Scrape this out of the dbt metadata
2. Construct a requirements.txt file
3. Install the packages to a virtual env
4. Zip up the installed packages then upload

Reply:
**[Praise]** I think that's a good place to start! It's nice that dbt already has a syntax for defining requirements.